packages/components/src/QuestionToolTip.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import styled from "styled-components";
+import { colors } from "./styleConstants";
+
+export interface QuestionToolTipProps {
+  explainerText: React.ReactNode;
+}
+
+export interface QuestionToolTipState {
+  open: boolean;
+}
+
+export interface TipProps {
+  bottom: number;
+  left: number;
+}
+
+const Tip = styled<TipProps, "div">("div")`
+  position: absolute;
+  bottom: ${(props: TipProps) => props.bottom}px;
+  left: ${(props: TipProps) => props.left}px;
+  margin-left: -90px;
+  width: 180px;
+  color: ${colors.basic.WHITE};
+  background: rgba(21, 21, 21, 0.9);
+  text-align: center;
+  border-radius: 3px;
+  padding: 13px;
+  font-size: 12px;
+  &:after {
+    content: "";
+    position: absolute;
+    left: calc(50% - 5px);
+    top: 100%;
+    width: 0;
+    height: 0;
+    border-left: 5px solid transparent;
+    border-right: 5px solid transparent;
+    border-top: 10px solid rgba(21, 21, 21, 0.9);
+  }
+`;
+
+const Wrapper = styled.div`
+  width: 18px;
+  height: 18px;
+  position: relative;
+`;
+
+const HitBox = styled.div`
+  position: absolute;
+  width: 180px;
+  height: 50px;
+  bottom: 0;
+  left: -81px;
+  border-radius: 50%;
+  background-color: transparent;
+`;
+
+export class QuestionToolTip extends React.Component<QuestionToolTipProps, QuestionToolTipState> {
+  public divEl: HTMLDivElement | null;
+  public bucket: HTMLDivElement = document.createElement("div");
+
+  constructor(props: QuestionToolTipProps) {
+    super(props);
+    this.divEl = null;
+    this.state = {
+      open: false,
+    };
+  }
+
+  public componentDidMount(): void {
```

Comment:
Do we need to worry about removing `bucket` when the component unmounts?

Reply:
yeah, i guess thats a good thing to do